test/e2e/certificates_test.go

```diff
@@ -71,16 +72,20 @@ var _ = Describe("ACME Certificate", Ordered, func() {
 				certManagerCAInjectorDeploymentControllerName},
 			validOperatorStatusConditions)
 		Expect(err).NotTo(HaveOccurred(), "Operator is expected to be available")
+
+		By("creating a test namespace")
```

Comment:
would refrain from using a common namespace that is re-used for every test case, this can have unknown side-effects (for eg. challenge(s) for the same domain getting re-used or other ripple effects).

The one namespace per test was introduced to ensure that we can run tests in parallel whenever we want to do that. I see that irrespective of using the central ns for testing your implementation can still use the dump events logic well, so shouldn't be a big issue it seems.

Reply:
My absolute bad, it's in `BeforeEach()` and not `BeforeAll()`;
please disregard my earlier comment.